cSpell.json

```diff
@@ -0,0 +1,81 @@
+{
+    "version": "0.2",
+    "language": "en",
+    "words": [
+        "browsable",
+        "Doomen",
+        "formattable",
+        "Guids",
+        "LINQ",
+        "parameterless",
+        "refactorings",
+        "struct"
+    ],
+    "ignoreWords": [
+        "Gallio"
+    ],
```

Comment:
is this still needed when `docs/_pages/releases.md` is ignored?

Reply:
True